### ISSUE TITLE
layer issue with bushes cutting top of player's head off

### ISSUE THREAD

**Issue by dantogno** [dantogno/FishStory#164]
![image](https://user-images.githubusercontent.com/3150710/74111124-e423a280-4b57-11ea-9b93-b3cf4ce08243.png)

**Comment by Cabrill** [comment]
I'd like @vchelaru to confirm, but I'm pretty sure this can't be fixed without either shrinking the character or extending the collision just enough that the player can't get close enough to overlap because the `Position.Z` is intentionally set higher than the player's so they are able to walk behind it.

**Comment by vchelaru** [comment]
Yes, @Cabrill is right - we either have to split the layers and extend the collision or...I dunno.
